lib/user.py

```diff
@@ -8,9 +8,11 @@
 import base64
 import uuid
 import re
+from lib.document import Document
 from collections import defaultdict
 from responseprinter import ResponsePrinter
 
+PRINTER = ResponsePrinter()
```

Comment:
@tleyden fixed. Also changed the name as it is more than a "response" printer now